### PR TITLE
feat: agent install prompt & `nci`

### PR DESCRIPTION
**Pull request by privatenumber**
As per https://github.com/antfu/ni/pull/5

Here's the agent install prompt feature.

`nci` PR https://github.com/privatenumber/ni/pull/2

### CODE REVIEW

File: src/detect.ts [hunk 1]
```diff
@@ -9,10 +11,22 @@ export async function detect() {
...
   if (agent && !cmdExists(agent)) {
-    const url = INSTALL_PAGE[agent]
-    const link = terminalLink(url, url)
-    console.log(`Detected ${agent} but it doesn't seem to be installed.\nFollow the instructions to install ${agent}: ${link}`)
-    process.exit(1)
+    if (process.env.CI)
+      throw new Error(`Detected ${agent} but it doesn't seem to be installed`)
+
+    const link = terminalLink(agent, INSTALL_PAGE[agent])
+    console.log(`Detected ${link} but it doesn't seem to be installed.\n`)
```
Comment (antfu) [inline src/detect.ts:18]:
```suggestion
    console.warn(`Detected ${agent} but it doesn't seem to be installed.\n`)
    if (process.env.CI)
      process.exit(1)

    const link = terminalLink(agent, INSTALL_PAGE[agent])
```

WDYT?

**Comment by antfu** [comment]
Thanks! The `nci` looks good too and let's have it ;)

File: src/detect.ts (hunk 1 above)
Comment (privatenumber) [reply src/detect.ts:18]: Good call! Showing the JS stack here is pretty useless

**Comment by privatenumber** [comment]
Just merged nci in

Thanks @antfu

**Comment by antfu** [comment]
Thanks!

**Pull request merged**